workflow/cases/coupled_free_forecast_wave_exp1.yaml

```diff
@@ -0,0 +1,60 @@
+case:
+  places:
+    workflow_file: layout/free_forecast_gfs.yaml
+    #BASE_CPLIC: /scratch2/NCEPDEV/climate/climpara/S2S/IC
+
+  settings: 
+    SDATE: 2013-04-01t00:00:00
+    EDATE: 2013-04-01t00:00:00
+
+    cplwav: .true.
+    cplwav2atm: .true.
+    cplflx: .true.
+    print_esmf: .true.
+    nems_temp: 'med_atm_ocn_ice_wav'
+    nems_temp_cold: 'medcold_atm_ocn_ice_wav'
+    mom6ic_prepared: .false.
+    KEEPDATA: YES
```

Comment:
Is this for the prototypes?   If so can we name this p5 instead of exp1? Also if this is for the prototypes KEEPDATA should be NO,

Reply:
Should we also update the README as part of this PR for the additional case files to explain the additional ones?

Reply:
OK let me address it. I will name it p5